app/frontend/src/features/userPage/UserPage.tsx

```diff
@@ -1,11 +1,13 @@
-import { Grid, makeStyles } from "@material-ui/core";
-import React from "react";
+import { Grid, makeStyles, Typography } from "@material-ui/core";
```

Comment:
This file is more of a placeholder to keep everything in this branch working, will likely be deleted before this gets merged due to Andrea's work.